satpy/tests/test_modifiers.py

```diff
@@ -392,6 +392,104 @@ def test_compositor(self, calculator, apply_modifier_info, sza):
                                       masking_limit=NIRReflectance.MASKING_LIMIT)
 
 
+class TestPSPRayleighReflectance(unittest.TestCase):
```

Comment:
What do you think about moving this test to a new module in `tests/modifier_tests/test_atmosphere.py`? Additionally, could you make this class not based on `unittest.TestCase`? That will make it a "pytest" class and should make your parametrize functionality work.